sdcm/nemesis.py

```diff
@@ -1434,8 +1434,21 @@ def disrupt_kill_scylla(self):
     def disrupt_no_corrupt_repair(self):
         # prepare test tables and fill test data
         for i in range(10):
+            ks_name = f"drop_table_during_repair_ks_{i}"
+            test_keyspaces = self.cluster.get_test_keyspaces()
+
+            # If a keyspace doesn't exist, create it
+            if ks_name not in test_keyspaces:
+                self.log.debug("Going to create a new keyspace `%s'...", ks_name)
+                self.tester.create_keyspace(
```

Comment:
Check the default timeout inside this implemention, we might want to enlarge it to some degree,
So we won't timeout on those calls

Reply:
Why not just run "CREATE KEYSPACE IF NOT EXIST" ?

Reply:
> Check the default timeout inside this implementation, we might want to enlarge it to some degree, So we won't timeout on those calls
i don't see any timeout set in that function, so it should be implicit somewhere...

> Why not just run "CREATE KEYSPACE IF NOT EXIST" ?
this is exactly what it is doing underneath, but we still need to create the tables

Reply:
> > Check the default timeout inside this implementation, we might want to enlarge it to some degree, So we won't timeout on those calls
> i don't see any timeout set in that function, so it should be implicit somewhere...
> 
> > Why not just run "CREATE KEYSPACE IF NOT EXIST" ?
> this is exactly what it is doing underneath, but we still need to create the tables

we could do  `IF NOT EXIST` for tables as well (as I've suggested, we should create the tables)
and for the little amount of data we put in, we might as well generate the data with the python code, and skip c-s.

using c-s in this case, was a quick way to generate new ks/table with small amount of data.
we can always to switch to a different method/tool, i.e. the tool in use isn't the important part (and we don't care about speed/performance in those)

Reply:
1. If we do "IF NOT EXIST" we don't need to query all tables and check that the table isn't there.

2. I don't mind to write the data without c-s, as long as it's quick enough.
In this case, since I don't want to start a saga of testing, the fix should be very simple.

For loop with create keyspace and create table, nothing else.

Reply:
Moved ks and table creation to `_prepare_test_table` method